newIDE/app/src/InAppTutorial/InAppTutorialStepDisplayer.js

```diff
@@ -24,7 +25,7 @@ const styles = {
     position: 'absolute',
     left: 20,
     bottom: 20,
-    zIndex: 5, // Scene editor mosaic z-index is 4
+    zIndex: aboveMaterialUiMaxZIndex, // Ensure it sits above everything.
```

Comment:
Could you give the reason? I guess it's because you want to be able to display the avatar when a dialog is opened, but it's not clear.

Reply:
Yes for instance on the first step, where the user has to either log in/create an account or create a leaderboard, it was useful not to highlight a button as they are too many paths to handle.
It is then useful to be displayed above the backdrop of modals and the drawers. I'll update the comment.